cmd/access/node_builder/access_node_builder.go

```diff
@@ -576,14 +580,6 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 		AdminCommand("read-execution-data", func(config *cmd.NodeConfig) commands.AdminCommand {
 			return stateSyncCommands.NewReadExecutionDataCommand(builder.ExecutionDataStore)
 		}).
-		Module("transactions and collections storage", func(node *cmd.NodeConfig) error {
```

Comment:
This is redundant. We could just use node.Storage.Transactions and node.Storage.Collections